Makefile

```diff
@@ -8,7 +8,7 @@ EMBED?="true"
 
 PALETTE_ENDPOINT?=""
 REGISTRATION_URL?=""
-DISK_SIZE?="50000M"
+DISK_SIZE?="80000M"
```

Comment:
should need to add a default empty `EDGE_HOST_TOKEN` and make sure it gets passed into the docker run commands, the entrypoint.sh change looks good but I dont think the value will ever get to it if you don't pass it in through the makefile